internal/generator/vector/conf_test.go

```diff
@@ -3,6 +3,7 @@ package vector
 import (
 	"encoding/json"
 	"fmt"
+	helpers2 "github.com/openshift/cluster-logging-operator/test/framework/unit"
```

Comment:
Should be used as `unit` below.

Reply:
i don't remember doing this but I'll fix it